### ISSUE TITLE
high cpu-consumption

### ISSUE THREAD

**Issue by johuhr** [Ximi1970/systray-x#8]
when I activate systray-x, thunderbird permanently uses one cpu at 100%. Thunderbird is hardly usable then. As soon as I deactivate systray-x, cpu consumption returns to normal.

My System:
Betriebssystem: openSUSE Leap 15.1
KDE-Plasma-Version: 5.18.3
KDE-Frameworks-Version: 5.68.0
Qt-Version: 5.14.1
Kernel-Version: 4.12.14-lp151.28.40-default
Art des Betriebssystems: 64-bit
Prozessoren: 8 × Intel® Core™ i7-7700K CPU @ 4.20GHz
Speicher: 31,3 GiB Arbeitsspeicher

**Comment by Ximi1970** [comment]
@johuhr and thunderbird version?

**Comment by johuhr** [comment]
Thunderbird is 68.5.0
I installed Qt "normally" from the opensuse repository https://download.opensuse.org/repositories/KDE:/Qt5/openSUSE_Leap_15.1

**Comment by Ximi1970** [comment]
Ah thats also possible :smile: I willl have a look.

**Comment by Ximi1970** [comment]
Looks like my OBS "test" package is only partially working. Did you compile it yourself?

**Comment by johuhr** [comment]
OBS test package? I compiled systray myself using your Makefile - if this is meant.

**Comment by Ximi1970** [comment]
I also got a repo for the add-on but I killed the package refactoring the release mechanics. Need to fix that first.

https://build.opensuse.org/package/show/home:Ximi1970:Mozilla:Add-ons/systray-x

https://download.opensuse.org/repositories/home:/Ximi1970:/Mozilla:/Add-ons/openSUSE_Leap_15.1/

**Comment by grbdevnull** [comment]
I installed from the bionic 18.04 amd64 .deb package from Release test 2 and am seeing similar behavior. With the systray-x plugin activated, Thunderbird's CPU usage was continually between 55% and 75%. The moment I deactivated the plugin, Thunderbird's CPU usage dropped to 10% or lower (mostly between 0% and 2%).

System info:
Xubuntu 18.04 (64 bit) 
Thunderbird 68.4.1 (64 bit)
Qt version: default repository (sudo apt install qt5-default)
Intel(R) Core(TM)2 Duo CPU P8700  @ 2.53GHz

Hopefully this info is helpful.
